### PR TITLE
Add code blocks to the 3rd chapter in ep3

### PR DESCRIPTION
**Pull request by bocharsky-bw**
Resolve `03-protected-visibility` in https://github.com/knpuniversity/oo/issues/1

### CODE REVIEW

File: knpu/episode3/protected-visibility.md [hunk 1]
```diff
@@ -12,7 +12,9 @@ function and paste it over here. And then just add '(rebel)' at the end. Now
 you may be thinking "guys, that's some serious code duplication...". Well you're
 absolutely right, and we'll get to fixing that!
 
-For now what we've got is pretty straightforward, so let's refresh and ....oh
+[[[ code('7db3c66c91') ]]]
```
Comment (weaverryan) [inline knpu/episode3/protected-visibility.md:13]: I like to include the code blocks as early as possible (code block first, explanation after). So, I'd move this up, right after `And then just add '(rebel)' at the end.`

File: knpu/episode3/protected-visibility.md [hunk 2]
```diff
@@ -34,7 +36,11 @@ of the class you're working in.
 Now, there is another designation between `private` and `public` which is called `protected`. 
 `Protected` works exactly like `private` except that subclasses can access it, so when we change
 it here the error goes away. Cool! Let's do a temporary fix for the error we're getting by making
-all of these things `protected`. Everything in our `RebelShip` file looks happy again so let's refresh.
```
Comment (weaverryan) [inline knpu/episode3/protected-visibility.md:37]: Maybe we should add a code block after line 26 above, after this sentence:

> The answer has to do with this word `private` in front of `$name`.

We would show the private `$name` property in the ship class

File: knpu/episode3/protected-visibility.md [hunk 3]
```diff
@@ -52,7 +60,9 @@ Back to the browser and refresh, and things still work!
 
 Let's temporarily add a new `private` function to `Ship` called `getSecretDoorCodeToTheDeathstar()`.
 Since only Empire ships should have access to this you can see why setting it as `private` makes sense.
-And let's return the secret code 'Ra1nb0ws'. 
+And let's return the secret code 'Ra1nb0ws':
+
+[[[ code('d3f3bf13b0') ]]]
 
 Over in `RebelShip` I should not be able to access this new function since we set it to `private`.
```
Comment (weaverryan) [inline knpu/episode3/protected-visibility.md]: We're missing a code block right here showing us using this new function from inside `RebelShip` (I know, not obvious until you can see the video!)

**Comment by weaverryan** [comment]
The actual code that you grabbed for the code blocks looks perfect: only showing what you need to, but giving context when it makes sense (e.g. the class + opening and closing curly braces around the method). Nice work!

**Comment by bocharsky-bw** [comment]
@weaverryan Thanks! I'll applied your comments soon.

**Pull request merged**